compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -32,6 +33,8 @@
 
 namespace mlir::iree_compiler::IREE::GPU {
 
+using CodeGenPipeline = IREE::Codegen::DispatchLoweringPassPipeline;
```

Comment:
```suggestion
using IREE::Codegen::DispatchLoweringPassPipeline;
```
I'd stick to the existing name instead of introducing a new one